cypress/e2e/login-page.test.js

```diff
@@ -1,27 +1,89 @@
-/// <reference types="cypress" />
+/// <reference path="../cypress.d.ts" />
 
-context("/", () => {
-  context("for guest user", () => {
+context("/auth/login", () => {
+  context("when user is not authenticated", () => {
     beforeEach(() => {
-      cy.intercept("GET", "/api/user", (req) => {
-        req.reply({
-          statusCode: 401,
-          body: { message: "no api key found" },
+      cy.mockPortalSettings("portal-auth-only");
+      cy.mockUserIdentity("guest");
+
+      cy.visit("/auth/login");
+    });
+
+    it("renders login form", () => {
+      cy.getByTestId("LoginForm").should("be.visible");
+    });
+
+    it("allows switching to sign up form", () => {
+      cy.getByTestId("SignUpLink").should("be.visible").click();
+
+      cy.location("pathname").should("eq", "/auth/registration");
+    });
```

Comment:
this is common mistake when working with integration tests :) 

https://docs.cypress.io/guides/references/best-practices#Creating-tiny-tests-with-a-single-assertion